services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -586,6 +589,7 @@ public VeniceControllerConfig(VeniceProperties props) {
     this.danglingTopicCleanupIntervalSeconds = props.getLong(CONTROLLER_DANGLING_TOPIC_CLEAN_UP_INTERVAL_SECOND, -1);
     this.danglingTopicOccurrenceThresholdForCleanup =
         props.getInt(CONTROLLER_DANGLING_TOPIC_OCCURRENCE_THRESHOLD_FOR_CLEANUP, 3);
+    this.defaultMaxRecordSizeBytes = props.getInt(CONTROLLER_DEFAULT_MAX_RECORD_SIZE_BYTES, -1);
```

Comment:
I think we should put a valid default, such as `100MB` instead of `-1` here, and maybe we should put a sanity check when loading the config to prevent negative values or the default value should be at least `1MB`.

Reply:
Changed default value to `100MB` and added check for `< 1 MB` to throw `VeniceException`.